ui/pages/swaps/smart-transaction-status/smart-transaction-status.js

```diff
@@ -368,7 +368,7 @@ export default function SmartTransactionStatusPage() {
         </Box>
         <Box
           marginTop={3}
-          className="smart-transaction-status__background-animation smart-transaction-status__background-animation--top"
+          className="smart-transaction-status smart-transaction-status"
```

Comment:
Not sure why this class is doubled, or why it's being used here as I thought `smart-transaction-status` was being used at the root element of this component.

Reply:
Good catch.. don't know why I didn't see that 🙈 

I made the change locally but can't test it until we get some more ETH in the swaps wallet